### PR TITLE
Update target directory for improve page button

### LINKED ISSUE #13: Improve this page links in footer broken
@iMichaela reported that the Hugo theme directives that generate links to open the relevant GitHub file for edit is broken after the migration and still points to usnistgov/OSCAL. We need to correct this now that we have migrated.

### PR DESCRIPTION
**Pull request by aj-stein-nist**
Adjust Hugo configuration to point updated "Improve this page" link on the footer of website, closes #13.

### CODE REVIEW

File: src/config.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ params:
   header:
     project_shortname: "OSCAL"
     project_name: "OSCAL: the Open Security Controls Assessment Language"
-  page_repo_path: https://github.com/usnistgov/OSCAL/tree/main/docs/content
+  page_repo_path: https://github.com/usnistgov/OSCAL-Pages/tree/main/src/content
```
Comment (iMichaela) [inline src/config.yaml:34]: I wonder if the external contributors should push the proposed updates to main or to develop branch...

**Review by iMichaela — COMMENTED** [review]
Should we have an ADR that documents to which branch the contributions should be PR-ed to?

File: src/config.yaml (hunk 1 above)
Comment (aj-stein-nist) [reply src/config.yaml:34]: Good question but this only controls when they click what they see to edit, not how it gets saved. Once they choose to save that commit they have to choose to save to a branch for a PR, they will be unable to commit to branch, they will be forced to do so. GitHub's interface controls that, not me.

Comment (aj-stein-nist) [reply src/config.yaml:34]: When you and Chris discussed this in review, I realized even if my answer stands your implied question about the target branch to change is worth calling out, I am updating my PR to point the `develop` branch not `main`.

```suggestion
  page_repo_path: https://github.com/usnistgov/OSCAL-Pages/tree/develop/src/content
```

**Pull request merged**